hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -139,7 +139,8 @@ const (
 	jobHostedClusterNameLabel      = "hypershift.openshift.io/cluster-name"
 	jobHostedClusterNamespaceLabel = "hypershift.openshift.io/cluster-namespace"
 
-	etcdCheckRequeueInterval = 10 * time.Second
+	etcdCheckRequeueInterval          = 10 * time.Second
+	skipControlPlaneNamespaceDeletion = "hypershift.openshift.io/skip-delete-hosted-controlplane-namespace"
```

Comment:
typically we put these with other [api constants](https://github.com/openshift/hypershift/blob/ff5710e71806fb6e85c4bffdcde5b5ceb484a2c6/api/hypershift/v1beta1/hostedcluster_types.go#L28)

Reply:
Thanks Cesar! Moved it to that file. Is the naming alright?